easybuild/easyconfigs/p/Python/Python-2.7.12-intel-2016b.eb

```diff
@@ -18,7 +18,8 @@ dependencies = [
     ('libreadline', '6.3'),
     ('ncurses', '6.0'),
     ('SQLite', '3.13.0'),
-    ('Tk', '8.6.5'),
+    # We don't build Tkinter as it requires an X11 stack
+    # ('Tk', '8.6.5'),
```

Comment:
I know you do, but I still don't see how that's a blocker for this PR.

Currently, our Tk easyconfigs require to have X11 provided by the OS, whatever the versionsuffix or configure options say. We should definitely fix that, but this shouldn't block any other PRs?